src/plugins/azure/auth.ts

```diff
@@ -10,47 +10,37 @@ You should have received a copy of the GNU General Public License along with @p0
 **/
 import { print2 } from "../../drivers/stdio";
 import { exec } from "../../util";
-import { KnownError } from "./types";
-
-const knownLoginErrors: KnownError[] = [
```

Comment:
This mini-framework wasn't flexible enough and just made the error handling more complex, I removed it in favor of just being explicit about which errors are being handled by each function.